lib/routes/users.js

```diff
@@ -22,29 +22,13 @@ const database = require('../database');
  * @returns {object} - Configured Express Application
  */
 const routes = application => {
-  return new Promise(resolve => {
-    const app = application;
+  const app = application;
 
+  return new Promise(resolve => {
```

Comment:
Saw this all over the place at Chrome Dev Summit. Whenever you just need to resolve as a promise with a single value, we can do [`Promise.resolve()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/resolve). This might seriously simplify a bunch of our code. Not something we _need_ to do, but a thought.